jo-rdan/prop-lite
# bug(fixing signup form): fix signup form

#### What does this PR do?
- fixes the signup form
- adds a button to submit details with link to login

#### Description of Task to be completed
- A user should be able to signup using a form and on clicking the submit button, they are redirected to login page.

#### How should this be manually tested?
- start the server on vs code live server
- use the url `http://127.0.0.1:5500/UI/pages/signup.html.` to open the signup page
- enter details in field and click signup
- you should be successfully redirected to signin.html

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#167160946](https://www.pivotaltracker.com/story/show/167160946)

#### Screenshots (if appropriate)
Image of signup page
![image](https://user-images.githubusercontent.com/51836855/60874627-2b763900-a1ed-11e9-9812-6474b9712a37.png)
